packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -245,13 +253,15 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
 
     // Render SXA Rendering Variant
     if (renderingDefinition.params?.FieldNames) {
-      return componentFactory(
-        renderingDefinition.componentName,
-        renderingDefinition.params.FieldNames
-      );
+      return withErrorBoundary(
```

Comment:
Might be better to add the Error Boundary outside of this function. Would avoid duplicate code (and let it focus solely on resolving the component from the component factory). In `getComponentsForRenderingData` around line 218, for example.